docker/dev_env/Dockerfile

```diff
@@ -42,16 +45,18 @@ RUN dnf -y install dnf-plugins-core \
     gcc \
     just \
     which \
-    pipx \
-    nodejs \
+    nodejs npm \
+    python3.12 pipx \
```

Comment:
I had removed `npm` (as it comes automatically with `nodejs`) and `python3.12` (as it is preinstalled on the system). I don't mind having them listed here but they don't really add anything.

Reply:
I figured that was why you removed them, I wanted to keep them as explicit dependencies for clarity of intention.